lib/features/home/data/property_item_data.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:equatable/equatable.dart';
+import 'package:hatspace/data/property_data.dart';
+import 'package:intl/intl.dart';
+
+class PropertyItemData extends Equatable {
+  final String? id;
+  final List<String> photos;
+  final String price;
+  final String name;
+  final PropertyTypes type;
+  final int bedrooms;
+  final int bathrooms;
+  final int parkings;
+  final int todayViews;
+  final String availableDate;
+  final String ownerName;
+  final String ownerAvatar;
+  final bool isFavorited;
+
+  const PropertyItemData(
+      {required this.id,
+      required this.photos,
+      required this.price,
+      required this.name,
+      required this.type,
+      required this.bedrooms,
+      required this.bathrooms,
+      required this.parkings,
+      required this.todayViews,
+      required this.availableDate,
+      required this.ownerName,
+      required this.ownerAvatar,
+      required this.isFavorited});
+
+  factory PropertyItemData.fromModel(Property property, User user) =>
+      PropertyItemData(
+          id: property.id,
+          photos: property.photos,
+          price: '${property.price.currency.symbol}${property.price.rentPrice}',
```

Comment:
This is UI logic, and we can use ICU format to use localisation string directly on UI

Reply:
@suesitran Can you explain more details ? 
define a new word in intl like "price": "{symbol}{price}" ? OR 
"price": "${price}", then for new country we will change the symbol ? For example
en : "price": "${price}"
vi : "price": "{price}đ" ? 

With the second method, we don't need to specify symbol in enum Currency no more right ?

Reply:
`{symbol}{price}`

Remember to format number display for `price`